reactor-core/src/test/java/reactor/test/LoggerUtils.java

```diff
@@ -62,7 +68,7 @@ public String getName() {
 
 		@Override
 		public boolean isTraceEnabled() {
-			return delegate.isTraceEnabled();
+			return delegate.isTraceEnabled() || (LoggerUtils.testLogger != null && LoggerUtils.testLogger.isTraceEnabled());
```

Comment:
for the condition, capture the `testLogger` into a local variable (the `testLogger` could be nulled out between the `!= null` test and the `isTraceEnabled()` call)